firebase-firestore/src/androidMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -325,6 +325,8 @@ actual class DocumentReference(val android: com.google.firebase.firestore.Docume
         }
         awaitClose { listener.remove() }
     }
+
+    actual companion object
```

Comment:
a declaration is required if I want to add extension methods on the companion object